### PR TITLE
Create ysalo.json

### PR DESCRIPTION
**Pull request by ysalo**

### CODE REVIEW

File: _pins/ysalo.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+gitHubHandle: ysalo
```
Comment (jr-cologne) [inline _pins/ysalo.json]: It's ``githubHandle``

File: _pins/ysalo.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude: 47.322322
+Longitude: -122.310682
```
Comment (jr-cologne) [inline _pins/ysalo.json]: the ``L`` of ``Longitude`` has to be small

File: _pins/ysalo.json (hunk 1 above)
Comment (africak) [inline _pins/ysalo.json]: :+1:

File: _pins/ysalo.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+gitHubHandle: ysalo
+latitude: 47.322322
```
Comment (africak) [inline _pins/ysalo.json]: lower case `l` for `latitude` :boat:

**Pull request merged**